assets/js/components/SapSystemDetails/tableConfigs.jsx

```diff
@@ -1,29 +1,101 @@
 import React from 'react';
+import classNames from 'classnames';
 
 import HostLink from '@components/HostLink';
 import ProviderLabel from '@components/ProviderLabel';
+import CleanUpButton from '@components/CleanUpButton';
+import Tooltip from '@components/Tooltip';
+import HealthIcon from '@components/Health';
 
 import Features from './Features';
 import InstanceStatus from './InstanceStatus';
 
 export const systemInstancesTableConfiguration = {
   usePadding: false,
+  rowClassName: 'bg-jungle-green',
   columns: [
-    { title: 'Hostname', key: 'instance_hostname' },
-    { title: 'Instance Number', key: 'instance_number' },
+    {
+      title: 'Hostname',
+      key: 'instance_hostname',
+      render: (content, item) => {
+        if (item.absent_at) {
+          return (
+            <Tooltip
+              content="Registered instance not found."
+              place="bottom"
+              isEnabled={!!item.absent_at}
+            >
+              <HealthIcon health="absent" />
+              <span className="mx-1 text-gray-600">{content}</span>
+            </Tooltip>
+          );
+        }
+        return <span>{content}</span>;
+      },
+    },
+    {
+      title: 'Instance nr',
+      key: 'instance_number',
+      render: (content, item) => (
+        <span className={classNames({ 'text-gray-600': !!item.absent_at })}>
```

Comment:
I'm not sure if there is any other way for calculating the class, but in any case, could we have some generic `render` function for this as it is repeated many times here?
```
const cellRender = (content, item) => (        <span className={classNames({ 'text-gray-600': !!item.absent_at })}>{content}</span>)```